build/DalamudBuild.cs

```diff
@@ -159,20 +163,32 @@ public class DalamudBuild : NukeBuild
                 .SetConfiguration(Configuration));
         });
 
+    Target SetCustomLogging => _ => _
```

Comment:
Please call this `SetCILogging`, since this should only really be set in the CI process.

Reply:
Will make this change!